src/Board.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Board.hpp"
+
+Board::Board(int row_number, int col_number,
```

Comment:
1 param per line is the best formatting, so please move col_number to the next line